javascript.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+
+function menuChange(e) {
```

Comment:
Also, when I click on the nav link in mobile view, it should close the menu and take me to the correct part of the page to which it is linked. Consider fixing it.

Reply:
Updated the link href.